### ISSUE TITLE
Veyon AD Auth non domain workstation

### ISSUE THREAD

**Issue by Shadowfita** [veyon/veyon#456]
Hi,

I have my work laptop which is not connected to a domain. When running Veyon even when AD authentication is set up in the settings it only asks me to login with my computers user and pass rather than authenticating with AD which causes my connection to be rejected when trying to view/control workstations that are connected to the domain. 

Is there a setting I am missing or is this not a feature (yet)?

Thanks.

**Comment by tobydox** [comment]
You should be able to authenticate as AD user by simply adding the domain to the username such as `MYDOMAIN\User` in the Veyon logon dialog. Have you tried that already?

**Comment by Shadowfita** [comment]
@tobydox  Yep! Thanks for the response. I've tried DOMAIN\User and I just receive incorrect username or password. The machine I am using isn't joined to the domain itself but I have specified (and successfully "tested" the AD connection) in the settings.

**Comment by mmeyer56** [comment]
I also tried it with the same problem. In the Veyon configurator I think that's the choice of the mode used for authentication. A valid LDAP configuration do not necessary permit a connection to clients. These clients cannot valid request from master if this computer is not a domain member. If this possibility exist I like this for administrator supervision to control computer over several domains.

**Comment by zdralic** [comment]
Did anyone solve this "issue", or discovered another way around it?

**Comment by Shadowfita** [comment]
@zdralic I haven't found any workarounds.

**Comment by Neuch2k20** [comment]
Hello.
I just create the same account wich I use in the domain in my local account manager (my computer is not domain-joined).
In this account don't refer to the domain, and use the same password as the domain account.

When I launch Veyon Master, just use the domain account and it works...
[Local creation and domain account use](https://user-images.githubusercontent.com/73959039/98147339-ad9f6c00-1ecc-11eb-99cb-cae212bf6cbe.png)
